File: ts/wam/5264.ts
export const WamChatWallpaper = 5264 as const

export interface ChatWallpaperEvent {
  /** field 1, wire `chat_wallpaper_change_applied` */
  chatWallpaperChangeApplied?: boolean
  /** field 2, wire `chat_wallpaper_source` */
  chatWallpaperSource?: CHATWALLPAPERENTRYTYPE
  /** field 3, wire `chat_wallpaper_type` */
  chatWallpaperType?: CHATWALLPAPERTYPE
  /** field 4, wire `chat_wallpaper_visit` */
  chatWallpaperVisit?: boolean
  /** field 5, wire `appearance_type` */
  appearanceType?: DEVICEAPPEARANCETYPE
}

File: go/wam/5264.go
const WamChatWallpaper = 5264 // channel: regular

type ChatWallpaperEvent struct {
	ChatWallpaperChangeApplied bool `wam:"chat_wallpaper_change_applied"` // field 1
	ChatWallpaperSource CHATWALLPAPERENTRYTYPE `wam:"chat_wallpaper_source"` // field 2
	ChatWallpaperType CHATWALLPAPERTYPE `wam:"chat_wallpaper_type"` // field 3
	ChatWallpaperVisit bool `wam:"chat_wallpaper_visit"` // field 4
	AppearanceType DEVICEAPPEARANCETYPE `wam:"appearance_type"` // field 5
}